homeassistant/components/plugwise/climate.py

```diff
@@ -44,14 +47,16 @@
 DEFAULT_MAX_TEMP = 30
 
 # HVAC modes
-ATTR_HVAC_MODES = [HVAC_MODE_AUTO, HVAC_MODE_OFF]
+ATTR_HVAC_MODES_1 = [HVAC_MODE_HEAT, HVAC_MODE_AUTO]
```

Comment:
We don't name constants that are values `ATTR_`. Only attribute keys are named `ATTR_`.

Reply:
Updated accordingly.